src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -616,8 +616,10 @@ def __init__(
                     settings = _file["settings"]
                     rpvars = settings["Rampant Variables"][0]
                     rp_vars_str = rpvars.decode()
-            elif Path(case_file_name).match("*.cas") or Path(case_file_name).match(
-                "*.msh"
+            elif (
```

Comment:
@hpohekar I'm surprised to see chnges in the file read logic. Please do not end support for .msh files. The goal of this task is only to change the way we are storing the example data such that it's more efficient. cc: @prmukherj

Reply:
Done. Thank you.

Reply:
@seanpearsonuk We want to support all 3 *.msh, *.msh.h5 and *.msh.gz right ?

Reply:
@hpohekar (cc: @prmukherj)

Note that .h5 is already handled above. Ignore the .gz pattern here. In other words, no code changes - just focus on the data file removal. 

If we do happen to find a bug in this code, we should only address it properly with creation of a defect issue and a separate PR with code change plus new unit test(s). 

That approach should apply generally -- we should avoid bundling additional changes into PRs, which is a hazardous strategy. Note also that this code is very sensitive for optiSLang functionality.

Reply:
@seanpearsonuk Yes, right. Thanks a lot.